### DISCUSSION TITLE
[bug]missing the options for proof type

### DISCUSSION THREAD

**Discussion by hzy179** [community/community#76835] [GitHub Education]
### Hi everyone,

i can't find the  proof type when provide proof
{:class=>"discount_request_id", :data=>{:discount_request_id=>12649480}}
[imgur](https://imgur.com/5s1Pxxx)

**Comment by YourSonnet18** [comment]
same problem

**Reply by YourSonnet18** [reply]
It's been a while and the problem still exists. No strange code though, the proof type option is still missing.

**Comment by alihoushngi** [comment]
do you work on javascript?

**Comment by shangji666** [comment]
same proble. Can't find the select box
"Your application cannot be reviewed until you fix the following:
Please select proof type 'Other' for this image."

**Comment by dakshithadissanayaka** [comment]
It seems you're having trouble locating the proof type option when submitting proof for a discount request. The provided information indicates a discount request ID of 12649480 and an image hosting link (imgur). However, this doesn't directly address the issue of missing proof type options.

To effectively identify the root cause of the problem, it would be helpful to understand the specific context of the discount request. Please provide more details about the discount request, such as the platform or service where you're applying for the discount, the type of proof required, and any specific instructions or guidelines provided for submitting proof.

With more context, we can better assist in identifying the missing proof type option and suggest potential solutions.

**Comment by Begunner** [comment]
Here is my solution: naming the img "myStudentIdCard.jpeg" even if it isn't a real one. 
It works.

**Reply by YourSonnet18** [reply]
Damn it! it really works! Thank you!

**Reply by JustSunnyYing** [reply]
It really work! Thank you!
